mediaflow_proxy/extractors/factory.py

```
@@ -35,7 +37,8 @@ class ExtractorFactory:
         "LuluStream": LuluStreamExtractor,
         "DLHD": DLHDExtractor,
         "Vavoo": VavooExtractor,
-        "Fastream": FastreamExtractor
+        "Fastream": FastreamExtractor,
+        "Voe": VoeExtractor
     }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)